### PR TITLE
docs: basics-section

### PR DESCRIPTION
**Pull request by DaemonLoki**

### CODE REVIEW

File: packages/react-sdk/docusaurus/docs/React/01-basics/04-installation.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+yarn add @stream-io/video-react-sdk
+```
+
+For `npm`:
+
+```
+npm install @stream-io/video-react-sdk
+```
+
+### Install the low-level client
```
Comment (oliverlaz) [inline packages/react-sdk/docusaurus/docs/React/01-basics/04-installation.mdx:22]: I think we should mention the `react-bindings` here. They are implicit within the SDK but, if one wants to build their app on top of our LLC, they can come quite handy!

- `yarn add @stream-io/video-react-bindings` and `npm install @stream-io/video-react-bindings`

File: packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx [hunk 2]
```diff
@@ -3,4 +3,86 @@ id: 07-ui-customization-options
...
+- [Icons and Colors](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_variables.scss)
+
+### Using low-level client combined with custom UI
+
+If you want to fully customize the experience, you can also use the client SDK to access the logic and data from the Video SDK and fully customize the UI.
+
+Here is an example of how you can use the `useParticipants()` hook to get all call participants and render them in your application.
+
+```tsx
+import { useParticipants } from '@stream-io/video-react-bindings';
```
Comment (oliverlaz) [inline packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx:64]:
```suggestion
import { useParticipants } from '@stream-io/video-react-sdk';
```

I'd primarily advise to use the SDK-provided hooks.

File: packages/react-sdk/docusaurus/docs/React/01-basics/04-installation.mdx (hunk 1 above)
Comment (DaemonLoki) [reply packages/react-sdk/docusaurus/docs/React/01-basics/04-installation.mdx:22]: Added, with a quick explanation.

File: packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx (hunk 2 above)
Comment (DaemonLoki) [reply packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx]: Corrected the import

**Review by oliverlaz — APPROVED** [review]

**Pull request merged**